torchsynth/globals.py

```diff
@@ -29,6 +30,11 @@ def __init__(
         self.sample_rate = sample_rate
         self.buffer_size = buffer_size
 
+    def to(self, device: torch.device):
+        self.batch_size = self.batch_size.to(device)
+        self.sample_rate = self.sample_rate.to(device)
+        self.buffer_size = self.buffer_size.to(device)
```

Comment:
Nice

Reply:
Only thing I don't like is usually .to returns a copy and doesn't modify stuff in place